.gitignore

```diff
@@ -146,5 +146,14 @@ genre_prediction/data/statistics.csv
 lmd_full
 data
 
+# model files 
+models/model_2048_10kseq/pytorch_model.bin
+models/model_2048_10kseq/tokenizer_config.json
+models/model_2048_10kseq/special_tokens_map.json
+models/model_2048_10kseq/config.json
+models/model_2048_10kseq/generated_sequences/0699f83b5e4ff9ffb292e465757a1e37ec6ce74d98a049c8c5cbfa03e08bc271.txt
```

Comment:
hmm not sure if it is a good idea to add hash stuff here:D maybe whole folder directly?